addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -1,16 +1,40 @@
 import { Record } from "@mail/core/common/record";
+import { Deferred } from "@web/core/utils/concurrency";
+
+const ASYNC_GETTER_TIMEOUT = 120_000;
```

Comment:
There is not a big value to have a separate constant for a timeout used in a single place.